crates/router/tests/connectors/multisafepay.rs

```diff
@@ -370,16 +403,17 @@ async fn should_fail_void_payment_for_auto_capture() {
 #[actix_web::test]
 async fn should_fail_capture_for_invalid_payment() {
     let capture_response = CONNECTOR
-        .capture_payment("123456789".to_string(), None, None)
+        .capture_payment("123456789".to_string(), None, get_default_payment_info())
         .await
         .unwrap();
     assert_eq!(
         capture_response.response.unwrap_err().message,
-        String::from("No such payment_intent: '123456789'")
+        String::from("Something went wrong")
```

Comment:
this testcase was not working before? how the error message changed now?

Reply:
@jagan-jaya No it wasn't working. This is default value from the template code